File: dataset_reduzido/codenet/Python/Code/s052231578.py
# B - AtCoder Beginner Contest 111

import bisect

N = int(input())
arr = [111, 222, 333, 444, 555, 666, 777, 888, 999]
ans_idx = bisect.bisect_left(arr, N)
print(arr[ans_idx])

File: output/Llama-2-100/codenet/Python/Java/s052231578.java
# B - AtCoder Beginner Contest 111

import java.util.Arrays;

N = sc.nextInt();
arr = new int[] {111, 222, 333, 444, 555, 666, 777, 888, 999};
int ans_idx = Arrays.asList(arr).binarySearch(N);
System.out.println(arr[ans_idx]);
